fe/fe-core/src/main/java/com/starrocks/statistic/BasicStatsMeta.java

```diff
@@ -190,4 +205,50 @@ public boolean isUpdatedAfterLoad(LocalDateTime loadTime) {
             return updateTime.isAfter(loadTime);
         }
     }
+
+    public void setProperties(Map<String, String> properties) {
+        this.properties = properties;
+    }
+
+    public void setUpdateTime(LocalDateTime updateTime) {
+        this.updateTime = updateTime;
+    }
+
+    public void setAnalyzeType(StatsConstants.AnalyzeType analyzeType) {
+        this.type = analyzeType;
+    }
+
+    public Map<String, ColumnStatsMeta> getAnalyzedColumns() {
+        Map<String, ColumnStatsMeta> deduplicate = Maps.newHashMap();
+        // TODO: just for compatible, we can remove it at next version
+        for (String column : ListUtils.emptyIfNull(columns)) {
+            deduplicate.put(column, new ColumnStatsMeta(column, type, updateTime));
+        }
+        deduplicate.putAll(columnStatsMetaMap);
+        return deduplicate;
+    }
+
+    public String getColumnStatsString() {
+        if (MapUtils.isEmpty(columnStatsMetaMap)) {
+            return "";
+        }
+        return columnStatsMetaMap.values().stream()
+                .map(ColumnStatsMeta::simpleString).collect(Collectors.joining(","));
+    }
+
+    public void addColumnStatsMeta(ColumnStatsMeta columnStatsMeta) {
+        this.columnStatsMetaMap.put(columnStatsMeta.getColumnName(), columnStatsMeta);
+    }
+
+    @Override
+    public void gsonPostProcess() throws IOException {
```

Comment:
Better to use default constructor to BasicStatsMeta(), the default value will be set.

Reply:
done